File: 580a/main.go
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	flag := 0
	max := 1

	var cur int32
	var prev int32
	fmt.Scan(&prev)

	for i := 1; i < n; i++ {
		fmt.Scan(&cur)
		if cur < prev {
			if max < i-flag {
				max = i - flag
			}
			flag = i
		}
		prev = cur
	}

	if max < (n - flag) {
		max = n - flag
	}
	fmt.Print(max)
}

File: 580a/main.cpp
#include <stdlib.h>
#include <stdio.h>
#include <iostream>
using namespace std;

int main() {
    int n;
    cin>>n;

    int i = 1;
    int flag = 0;
    int max = 1;
    int cur;
    int prev;
    cin >> prev;

    for (; i < n; i++) {
        cin >> cur;
        if (cur < prev) {
            if (max < i - flag) {
                max = i - flag;
            }
            flag = i;
        }
        prev = cur;
    }

    if (max < n-flag) {
        max = n - flag;
    }
    cout << max;
}
